### ISSUE TITLE
Morgana skills based on magic not spr

### ISSUE THREAD

**Issue by sardukar922** [lyrgard/ffbeEquip#374]
Morgana skills are summon damage that is 100% magic based, not spirit based.

**Comment by lyrgard** [comment]
Yes, and that's what the builder should build her for. What is the problem
exactly?

Le jeu. 18 juil. 2019 à 20:53, sardukar922 <notifications@github.com> a
écrit :

**Comment by sardukar922** [comment]
ok i did some more digging and it it correct the way you have it.  I was thinking there were more magic based evo equipment or abilities. i guess there isnt yet, mosdtly spr equip.

**Comment by Darwe-Canine** [comment]
This issue can be closed.
